src/TMedia.cpp

```diff
@@ -1033,9 +1090,9 @@ void TMedia::handlePlayerPlaybackStateChanged(QMediaPlayerPlaybackState playback
         TEvent mediaFinished{};
         mediaFinished.mArgumentList.append("sysMediaFinished");
 #if QT_VERSION < QT_VERSION_CHECK(6, 0, 0)
```

Comment:
In the medium term we can remove the Qt 5.x stuff - as it has already become dead code (with the recently added minimum Qt **6.4.0** version requirement but culling all those bits is not vital at this point...

Reply:
Yes, for now I will leave it here, that sounds like a PR for all of Mudlet

Reply:
Changed my mind and pulled out the qt5 code now